ruleset/sca/nginx/cis_nginx_1.yml

```diff
@@ -0,0 +1,868 @@
+    compliance:
+      - cis: ["2.2.1"]
+      - cis_csc: ["5.1"]
+    references:
+      - http://nginx.org/en/docs/ngx_core_module.html#user
+    condition: all
+    rules:
+      - 'f:/etc/nginx/nginx.conf -> r:^user\s+nginx;'
+      - 'c:sudo -l -U nginx -> r:^User nginx is not allowed to run sudo\.*'
+      - 'c:groups nginx -> r:^nginx\s:\snginx$'
+
+# 2.2.2 Ensure the NGINX service account is locked (Scored)
+  - id: 23009
+    title: "Ensure the NGINX service account is locked"
+    description: "The nginx user account should have a valid password, but the account should be locked. NOTE: If a different account is used to run nginx, that account's name should be substituted for nginx in the audit and remediation procedures."
+    rationale: "As a defense-in-depth measure, the nginx user account should be locked to prevent logins and to prevent someone from switching users to nginx using the password. In general, there shouldn't be a need for anyone to have to su as nginx, and when there is a need, sudo should be used instead, which would not require the nginx account password."
+    remediation: "Use the passwd command to lock the nginx service account: 'passwd -l nginx'"
+    compliance:
+      - cis: ["2.2.2"]
+      - cis_csc: ["5.1"]
+    condition: all
+    rules:
+      - 'c:passwd -S nginx -> r:^nginx L\.+'
+
+# 2.2.3 Ensure the NGINX service account has an invalid shell (Scored)
+  - id: 23010
+    title: "Ensure the NGINX service account has an invalid shell"
+    description: "The nginx account should not have the ability to log in, so the /sbin/nologin shell should be set for the account."
+    rationale: "The account used for nginx should only be used for the nginx service and does not need to have the ability to log in. This prevents an attacker who compromises the account to log in with it."
+    remediation: "Change the login shell for the nginx account to /sbin/nologin by using the following command: 'chsh -s /sbin/nologin nginx'"
+    compliance:
+      - cis: ["2.2.3"]
+      - cis_csc: ["5.1"]
+    condition: all
+    rules:
+      - 'c:grep nginx /etc/passwd -> r:/sbin/nologin'
+
+# 2.3 Permissions and Ownership
+
+# 2.3.1 Ensure NGINX directories and files are owned by root (Scored)
+  - id: 23011
+    title: "Ensure NGINX directories and files are owned by root"
+    description: "The owner and group of the /etc/nginx directory and its files should be root."
+    rationale: "Setting ownership to only those users in the root group and the root user will reduce the likelihood of unauthorized modifications to the nginx configuration files."
+    remediation: "Run the following command to ensure ownership and group ownership is set to root: 'chown -R root:root /etc/nginx'"
+    compliance:
+      - cis: ["2.3.1"]
+      - cis_csc: ["5.1"]
+    condition: all
+    rules:
+      - 'c:stat /etc/nginx | grep "Access: (" -> r:Uid:\s+\(\.+root\)\s+Gid:\s+\(\.+root\)$'
```

Comment:
Please, add -L option to stat command like:
`       - 'c:stat -L /etc/nginx | grep "Access: (" -> r:Uid:\s+\(\.+root\)\s+Gid:\s+\(\.+root\)$'`

Reply:
Added in this commit: https://github.com/wazuh/wazuh/pull/8814/commits/6ec97103ff5497f488ddfd1b6f8ce062129c48f4